content/jobs/README.md

```diff
@@ -0,0 +1,46 @@
+# Job Post Instructions
+
+---
+
+Hey there! If you're a local-ish tech company looking to hire talent we at NewHaven.io would love to help out by putting some information about your organization and open role(s) on our website.
+
+Just clone this repository, add your information to [this page](https://github.com/newhavenio/newhavenio/tree/master/content/jobs/jobs.yaml), and submit a pull request.
+
+```markdown
+- company: "NewHaven.IO"
+  jobURL: "https://newhaven.io"
+  jobTitle: "Community Moderator"
+  location: "New Haven, CT // Remote"
+  tech: "Slack, MeetUp, GatsbyJS"
```

Comment:
Do we want to enumerate the tech here? Can the description suffice?

Reply:
Eh, easier for someone looking for a web dev job to see if something has C++ at a quick glance.

Reply:
That's a valid point. So long as this is just for ease of presenting keywords and nothing we'd ever want to query in the future then 👍. User entered free form tagging can be shenanigans fast.

Reply:
That's the idea!